src/routes/routes.ts

```diff
@@ -56,7 +56,9 @@ const enrollmentDashboardRoutes = {
   EVENTS: 'events',
   CE_ASSESSMENTS: 'ce-assessments',
   CUSTOM_CASE_NOTES: 'case-notes',
-  ASSESSMENT: 'assessments/:formRole/:assessmentId?',
+  ASSESSMENT: 'assessments/:formRole/:assessmentId?', // Legacy
+  VIEW_ASSESSMENT: 'assessments/:formRole/:assessmentId',
+  NEW_ASSESSMENT: 'assessments/:formRole/new/:formDefinitionId?',
```

Comment:
intake and exit: want to be able to just link to it regardless of whether it exists or not. act as shortcuts to redirect based on whether assmt exists or not

`/intake`
`/exit`


new assessment: `/assessments/new/:formDefinitionId`
existing assessment: `/assessments/:assessmentId`

specify `household: bool` on the definition(?), get rid of the "is household x" logic from the frontend

Reply:
@ttoomey I ended up implementing it like this:
```
INTAKE: 'intake', // Go to intake (render household view, or navigate to View/Create individual assessment appropriate)
  EXIT: 'exit', // Go to intake (render household view, or navigate to View/Create individual assessment as appropriate)
  VIEW_ASSESSMENT: 'assessments/:assessmentId', // View/Edit individual assessment
  NEW_ASSESSMENT: 'assessments/new/:formDefinitionId', // Create new individual assessment
```

I think what I have here all works, and seems slightly better.. I did split out the queries so we have a dedicated query for _assessment_ form definition lookup, since it is different from looking up other forms.

I haven't done self-review or serious QA yes, I will re request you after that, but feel free to look before. 🙇‍♀️